feature/gnoi/system/tests/supervisor_switchover_test/supervisor_switchover_test.go

```diff
@@ -81,6 +80,13 @@ func TestSupervisorSwitchover(t *testing.T) {
 	rpStandbyBeforeSwitch, rpActiveBeforeSwitch := findStandbyRP(t, dut, supervisors)
 	t.Logf("Detected rpStandby: %v, rpActive: %v", rpStandbyBeforeSwitch, rpActiveBeforeSwitch)
 
+	switchoverReady := dut.Telemetry().Component(rpActiveBeforeSwitch).SwitchoverReady()
+	switchoverReady.Await(t, 30*time.Minute, true)
```

Comment:
I am wondering if 30 mins is too much. As it may timeout the test or Guitar workflow when run in production.

Reply:
30 minutes is the current default reservation timeout. The test run timeout is longer.